features/scv/scvcli/src/main/java/org/opennms/features/scv/cli/ScvCli.java

```diff
@@ -72,12 +75,27 @@ private ScvCli() {
 
     public SecureCredentialsVault getSecureCredentialsVault() {
         if (secureCredentialsVault == null) {
-            secureCredentialsVault = new JCEKSSecureCredentialsVault(this.keystore, this.password);
+            secureCredentialsVault = new JCEKSSecureCredentialsVault(this.keystore, this.password, lookupKeyStoreType());
         }
 
         return secureCredentialsVault;
     }
 
+    private  String lookupKeyStoreType() {
+        String keyStoreType = SecureCredentialsVault.KeyStoreType.JCEKS.name();
+        Properties properties = new Properties();
+        try {
+            properties.load(ScvCli.class.getResourceAsStream("/scvcli-filtered.properties"));
+            String opennmsHome = properties.getProperty("install.dir");
+            Properties scvProps = SecureCredentialsVault.loadScvProperties(opennmsHome);
```

Comment:
If password is not specified on the command line, we can also derive it from `org.opennms.features.scv.jceks.key` from same system properties and fallback on lookupDefaultPassword() if it doesn't exist.